packages/@uppy/file-input/src/FileInput.tsx

```diff
@@ -1,14 +1,40 @@
-import { UIPlugin } from '@uppy/core'
+import { h, type ComponentChild } from 'preact'
+import type { CSSProperties } from 'preact/compat'
+
+import { UIPlugin, Uppy, type UIPluginOptions } from '@uppy/core'
 import toArray from '@uppy/utils/lib/toArray'
-import { h } from 'preact'
+import type { Body, Meta } from '@uppy/utils/lib/UppyFile'
 
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore We don't want TS to generate types for the package.json
 import packageJson from '../package.json'
-import locale from './locale.js'
+import locale from './locale.ts'
+
+
+interface FileInputOptions extends UIPluginOptions {
```

Comment:
```suggestion
export interface FileInputOpts extends UIPluginOptions {
```

Reply:
I tried to name it similar to all the other plugins that `extends UIPluginOptions` for consistency. They all use `*Options`, e.g. ScreenCaptureOptions, StatusBarOptions, ThumbnailOptions etc